terraform/context_apply_test.go

```diff
@@ -2698,10 +2694,7 @@ func TestContext2Apply_moduleOrphanInheritAlias(t *testing.T) {
 		t.Fatal("must call configure")
 	}
 
-	checkStateString(t, state, `
-module.child:
-  <no state>
-  `)
+	checkStateString(t, state, "")
```

Comment:
What's the difference between `""` and `"<no state>"` here? (I see `"<no state>"` used in the above example.)

Reply:
It depends on whether there is an empty root module or not, probably depending on when `prune` was called. There are a lot of tests that are relying on very specific behavior for their checks, so I didn't even attempt to go through and audit them all.

Reply:
Fair enough! We can iterate on this some more when we revamp the state to be more type-aware, I suppose.